[Interaction] Convert RoV Chapter 1 Missions

claywar opened the pull request:
<!-- remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm: -->
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

* Converts and adds additional messaging to RoV Chapter 1
* Adds Container:noAction() to Interaction Framework to handle cases where no other return is available

claywar committed:
Interaction - Mission: Rhapsodies of Vanadiel


Remove unnecessary require

claywar committed:
Interaction - Mission: Resonance

claywar committed:
Interaction - Mission: Emissary from the Seas

Adjust previous mission comments to reflect proper mission IDs

claywar committed:
Interaction - Mission: Set Free


Restore Mission Status after completing mission


Add missing requires


Remove assumptive event and replace with TODO comment

claywar committed:
Interaction - Mission: The Beginning

claywar committed:
Interaction - Mission: Flames of Prayer

claywar committed:
Interaction - Mission: The Path Untraveled


Add comment for Gilgamesh position

claywar committed:
Interaction - Mission: At Heaven's Door

claywar committed:
Interaction - Mission: The Lion's Roar

claywar committed:
Interaction - Mission: Eddies of Despair I

claywar committed:
Interaction - Add Action - Container:noAction()

Tested with, and modifies TOAU13 NM Spawn

claywar committed:
Interaction - Mission: A Land After Time

claywar committed:
Interaction - Mission: Fate's Call


Remove unneeded comment

claywar committed:
Interaction - Mission: What Lies Beyond

claywar committed:
Interaction - Mission: The Ties that Bind

claywar committed:
Interaction - Mission: Impurity

claywar committed:
Interaction - Mission: The Lost Avatar

Also fixes invalid ID variable in The Lion's Roar

Fix incorrect comment

claywar committed:
Interaction - Mission: Volto Oscuro

claywar committed:
Interaction - Mission: Ring My Bell

claywar committed:
QC: RoV Chapter 1 through M1-11


Path Untraveled Updates


Fix Syntax for 1-4


Add upper path parameter for M1-7


Add upper parameter for M1-8


Fix 1-8 Syntax error


Fix M1-9 spawn occupied message


Add upper options for RoV M1-10, M1-11

claywar committed:
Merge remote-tracking branch 'origin/base' into interaction/rov-chapter-1

claywar commented:
Should be good to go at this point to review.

zach2good commented on scripts/missions/rov/1_05_The_Beginning.lua:
```diff
@@ -74,7 +74,7 @@ mission.sections =
         {
             ['_700']       = mission:event(276):setPriority(1005), -- RoV objectives are highest priority, set higher than progressEvent (1000)
-            ['Comitiolus'] = mission:event(6),
+            ['Comitiolus'] = mission:event(6), -- NPC Chat 12321, not event 6
```
What does this mean?

claywar replied:
Had the wrong item there and made a note to change it, but totally missed.  Replaced with the message received from caps.

zach2good commented on scripts/zones/Mhaura/npcs/Ekokoko.lua:
```diff
@@ -11,6 +11,8 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
+    -- TODO: This is not a random sequence, but 51 is displayed on first interaction ever,
```
Are you going to do this, or should we make an issue for this?

claywar replied:
Raised #1385

zach2good reviewed (approved):
Looks good 👍

zach2good commented:
Needs a rebase, apparently

claywar committed:
QC: RoV Chapter 1 Remaining Missions

Add dialogue change for M1-13

Add additional dialogue for M1-17 and DefaultActions for Yuhtunga qm


Add TODO comment for potential CS change


Modify remaining TODO for messageSpecial
